.github/workflows/ci.yml

```
@@ -129,6 +129,7 @@ jobs:
           cargo install cargo-supply-chain       || true
           cargo install cargo-unmaintained       || true
           cargo install group-runner             || true
+          npm install -g prettier
           go install github.com/rhysd/actionlint/cmd/actionlint@latest
```

```suggestion
          go install github.com/rhysd/actionlint/cmd/actionlint@latest
          npm install -g prettier
```
Let's please put this one at the bottom of this list.